src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+          fill="icon.alt"
+          as={BxFileArchiveSolid}
+        />
+      )
+    }
+    case "setting": {
+      return <Icon ml="0.75rem" fontSize="1.25rem" fill="icon.alt" as={BiCog} />
+    }
+    case "page": {
+      return (
+        <Icon
+          ml="0.75rem"
+          fontSize="1.25rem"
+          fill="icon.alt"
+          as={BiFileBlank}
+        />
+      )
+    }
+    case "image": {
+      return (
+        <Icon ml="0.75rem" fontSize="1.25rem" fill="icon.alt" as={BiImage} />
+      )
+    }
+    default: {
+      // NOTE: This is done to ensure exhaustive type matching
+      const error: never = iconType
+      throw new Error(`Unmatched fileType: ${error}`)
+    }
+  }
+}
+
+type FileType = "page" | "nav" | "setting" | "file" | "image"
+
+export interface EditedItemProps {
+  type: FileType[]
+  name: string
+  path: string[]
+  url: string
+  lastEditedBy: string
+  lastEditedTime: number
+}
+
+const ItemName = ({ name, path }: Pick<EditedItemProps, "name" | "path">) => {
+  const theme = useTheme()
+  return (
+    <VStack align="flex-start">
+      <Text textStyle="subhead-2" textColor="text.label" noOfLines={1}>
+        {name}
+      </Text>
+      <Breadcrumb
+        as={Text}
+        textOverflow="ellipsis"
+        overflowX="auto"
+        spacing="2px"
+        separator={<BiChevronRight color={theme.colors.text.helper} />}
+      >
+        {path.map((item) => {
+          return (
+            <BreadcrumbItem>
+              <Text textStyle="caption-2" textColor="text.helper">
+                {item}
+              </Text>
+            </BreadcrumbItem>
+          )
+        })}
+      </Breadcrumb>
+    </VStack>
+  )
+}
+
+type LastEditedMetaProps = Pick<
+  EditedItemProps,
+  "lastEditedBy" | "lastEditedTime"
+>
+const LastEditedMeta = ({
+  lastEditedBy,
+  lastEditedTime,
+}: LastEditedMetaProps): JSX.Element => {
+  const { date, time } = getDateTimeFromUnixTime(lastEditedTime)
+  return (
+    <HStack>
+      <Avatar
+        bg="primary.100"
+        name={lastEditedBy.slice(0, 2).split("").join(" ")}
+        textStyle="caption-2"
+        textColor="primary.400"
+        size="sm"
+      />
+      <VStack align="flex-start">
+        <Text textStyle="caption-2" textColor="text.helper">
+          {date}
+        </Text>
+        <Text textStyle="caption-2" textColor="text.helper">
+          {time}
+        </Text>
+      </VStack>
+    </HStack>
+  )
+}
+
+export interface RequestOverviewProps {
+  items: EditedItemProps[]
+}
+
+export const RequestOverview = ({
+  items,
+}: RequestOverviewProps): JSX.Element => {
+  const {
+    isExpanded,
+    inputRef,
+    handleExpansion,
+    handleCollapse,
+  } = useSearchbar({})
+  const columnHelper = createColumnHelper<EditedItemProps>()
+
+  const columns = [
+    columnHelper.display({
+      id: "filter",
+      header: () => (
+        <Th w="3rem">
+          <IconButton
+            aria-label="sort by file type"
```

Comment:
I think should change sort to filter here?

Reply:
yep, good catch!